tools/DevDiagnostics/DevHome.DevDiagnostics/Helpers/ImageOptionsInternalTool.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using DevHome.DevDiagnostics.Models;
+using DevHome.DevDiagnostics.Properties;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+
+namespace DevHome.DevDiagnostics.Helpers;
+
+public class ImageOptionsInternalTool : Tool
+{
+    private const string ButtonText = "\ue9d5";
+    private static readonly string _toolName = CommonHelper.GetLocalizedString("ImageOptionsName");
+
+    public ImageOptionsInternalTool()
+        : base(_toolName, ToolType.Unknown, Settings.Default.IsImageOptionsToolPinned)
+    {
+    }
+
+    public override IconElement GetIcon()
+    {
+        return new FontIcon
+        {
+            Glyph = ButtonText,
+            FontFamily = (FontFamily)Application.Current.Resources["SymbolThemeFontFamily"],
+        };
+    }
+
+    internal override void InvokeTool(ToolLaunchOptions options)
+    {
+        if (TargetAppData.Instance.TargetProcess == null || TargetAppData.Instance.TargetProcess.MainModule == null)
+        {
+            return;
+        }
+
+        try
+        {
+            FileInfo fileInfo = new FileInfo(Environment.ProcessPath ?? string.Empty);
+            Directory.SetCurrentDirectory(fileInfo.DirectoryName ?? string.Empty);
```

Comment:
>SetCurrentDirectory [](http://example.com/codeflow?start=22&length=19)

Not sure if you need to set the working directory. I do it for my utilities, but that's because I'm launching the utilities directly from my package location. In your case, you're using app execution aliases, and so I don't think this is needed.